addons/mail/static/src/components/message/message.xml

```diff
@@ -180,6 +180,9 @@
                             </ul>
                         </t>
                     </div>
+                    <t t-if="message.subject and threadViewer and threadViewer.thread and [env.messaging.inbox, env.messaging.history].includes(threadViewer.thread)">
+                        <p class="o_Message_subject">Subject: <t t-esc="message.subject"/></p>
+                    </t>
```

Comment:
Is it on purpose that the subject is below the content?

Reply:
+ I really think mailing channel should have the subject too, it's so confusing on them to post a message with a subject and then not see it

Reply:
> Is it on purpose that the subject is below the content?

I think it's below so that it doesn't affect to much overall style of message?

> + I really think mailing channel should have the subject too, it's so confusing on them to post a message with a subject and then not see it

Yes, this is clearly an issue... I have hard time deciding whether we should display it or not. Last year we also just wanted to remove this restriction and display whenever there is a subject, and it was too much noise. Maybe we should just relax to mailing channel? Hopefully no weird computation of subject field of message in case of a mailing channel originating from another channel... Or value (not) being language-dependent (bc record name is translatable field)?

Maybe some these issues already exist in inbox? Maybe this issue is diminished by "hiding" subject below message body? I think this too much of a hassle to tackle this issue right now, hence my stand to be conservative and do minimal effort to make it work while not introducing new issues.

Reply:
> > Is it on purpose that the subject is below the content?
> 
> I think it's below so that it doesn't affect to much overall style of message?
> 

That's how it's done in saas-13.3, so I did the same

Reply:
consensus: also display subject in mailing channels